### PR TITLE
[FEATURE]: checklist with members

### LINKED ISSUE #525: [FEATURE]:Checklist with members
## Summary

On the "Add Team Members" drawer, it should be render a checklist with all of the users. 

### Motivation

The checklist with all users will allow to check the users which will be part of the new team

### PR DESCRIPTION
**Pull request by CatiaAntunes96**
Relates to #525 

## Screenshots (if visual changes)
image

## Proposed Changes

  - On the "Add Team Members" drawer, the checklist is being shown.

This pull request closes #525

### CODE REVIEW

File: frontend/src/components/Teams/CreateTeam/ListMembers/index.tsx [hunk 1]
```diff
@@ -12,53 +16,68 @@ import {
...
-					submitBtnRef.current.click();
-				}
-			}
-		};
+	const saveMembers = () => {
+		const listOfUsers: User[] | undefined = [];
 
-		element?.addEventListener('keydown', keyDownHandler);
+		checked.forEach((id) => {
+			const userFound = users?.find((user) => user._id === id);
```
Comment (nunocaseiro) [inline frontend/src/components/Teams/CreateTeam/ListMembers/index.tsx:68]: The listOfUsers is not used after adding the elements. Will you use it in the next pull request?

File: frontend/src/components/Teams/CreateTeam/ListMembers/index.tsx [hunk 2]
```diff
@@ -12,53 +16,68 @@ import {
...
+	const setToastState = useSetRecoilState(toastState);
+
 	// References
+	const scrollRef = useRef<HTMLDivElement>(null);
 	const dialogContainerRef = useRef<HTMLSpanElement>(null);
-	const submitBtnRef = useRef<HTMLButtonElement | null>(null);
 
 	// Method to close dialog and reset switches state
 	const handleClose = () => {
+		setChecked([]);
```
Comment (nunocaseiro) [inline frontend/src/components/Teams/CreateTeam/ListMembers/index.tsx:42]: Why are you resetting the checked users? And what happens if you open the sidebar again? Do you need to check which members are already in the team? :)

File: frontend/src/components/Teams/CreateTeam/ListMembers/index.tsx (hunk 1 above)
Comment (CatiaAntunes96) [reply frontend/src/components/Teams/CreateTeam/ListMembers/index.tsx:68]: Yes, this array will be stored in a new atom, which will allow it to be shared among other components. It's in the issue 530.

File: frontend/src/components/Teams/CreateTeam/ListMembers/index.tsx (hunk 2 above)
Comment (CatiaAntunes96) [reply frontend/src/components/Teams/CreateTeam/ListMembers/index.tsx:42]: Well I thought that when the user clicks on the cancel button the main purpose would be cancel the whole operation, hence the reseting of the checked users. But, yes you are right, when we close the sidebar it should preserve the list. I will fix this

Comment (nunocaseiro) [reply frontend/src/components/Teams/CreateTeam/ListMembers/index.tsx:42]: I understand your point. If it's easier we can go with this. So when we open the modal we can check which members are already in the team and check them in the list

**Comment by nunocaseiro** [comment]
@all-contributors please add @CatiaAntunes96 for code, documentation and review

**Pull request merged**